components/CommentSection.jsx

```diff
@@ -0,0 +1,38 @@
+import Container from "@mui/material/Container";
+import { useOperation } from "react-openapi-client";
+import CircularProgress from "@mui/material/CircularProgress";
+import Box from "@mui/material/Box";
+import Comment from "./Comment";
+
+function CommentSection({ resourceId }) {
+  const { loading, data, error } = useOperation(
+    "getResourceComments",
+    resourceId
+  );
+
+  if (loading) {
+    return (
+      <Box>
+        <CircularProgress />
+      </Box>
+    );
+  }
+
+  if (error) {
+    return <div>Error: {error}</div>;
+  }
+
+  return (
+    <Container maxWidth="sm">
+      {data.map((comment) => (
+        <Comment
```

Comment:
Each comment needs a unique id, otherwise, an error is printed in the console

Reply:
done :)